backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -2,11 +2,15 @@ package cromwell.backend
 
 import akka.actor.Props
 import wdl4s.Call
+import wdl4s.expression.WdlStandardLibraryFunctions
 
 trait BackendLifecycleActorFactory {
   def workflowInitializationActorProps(workflowDescriptor: BackendWorkflowDescriptor,
                                        calls: Seq[Call],
                                        configurationDescriptor: BackendConfigurationDescriptor): Option[Props]
   def jobExecutionActorProps(jobDescriptor: BackendJobDescriptor, configurationDescriptor: BackendConfigurationDescriptor): Props
   def workflowFinalizationActorProps(): Option[Props]
+  def engineFunctions(workflowDescriptor: BackendWorkflowDescriptor,
```

Comment:
How about Expression Language Functions (`ELF`s)

(and that's the last time you make the mistake of asking me to name anything!)